dsc/item_submission.py

```diff
@@ -0,0 +1,30 @@
+import json
+import logging
+from dataclasses import dataclass
+from typing import Any
+
+from dsc.utilities.aws.s3 import S3Client
+
+logger = logging.getLogger(__name__)
+
+
+@dataclass
+class ItemSubmission:
+    """A class to store the required values for a DSpace submission."""
+
+    dspace_metadata: dict[str, Any]
+    bitstream_uris: list[str]
+    metadata_s3_key: str
+    metadata_uri: str = ""
```

Comment:
Minor, but maybe worth considering renaming `metadata_s3_uri` to match the pattern of `metadata_s3_key`.

Reply:
Updating to `metadata_s3_uri` and `bitstream_s3_uris` for clarity and consistency